### PR TITLE
MOSIP-32453

### PR DESCRIPTION
**Pull request by dhanendra06**

### CODE REVIEW

File: admin/kernel-masterdata-service/src/main/java/io/mosip/kernel/masterdata/service/impl/ZoneServiceImpl.java [hunk 1]
```diff
@@ -70,9 +68,6 @@ public class ZoneServiceImpl implements ZoneService {
 	@Autowired
 	ZoneRepository zoneRepository;
 
-	@Autowired
-	private ZoneUserService zoneUserService;
```
Comment (LoganathanSekar7627) [inline admin/kernel-masterdata-service/src/main/java/io/mosip/kernel/masterdata/service/impl/ZoneServiceImpl.java:74]: Is this used in this class?

File: admin/kernel-masterdata-service/src/main/java/io/mosip/kernel/masterdata/service/impl/DeviceServiceImpl.java [hunk 2]
```diff
@@ -107,9 +107,6 @@ public class DeviceServiceImpl implements DeviceService {
 	@Autowired
 	private PageUtils pageUtils;
 
-	@Autowired
-	private ZoneService zoneService;
```
Comment (LoganathanSekar7627) [inline admin/kernel-masterdata-service/src/main/java/io/mosip/kernel/masterdata/service/impl/DeviceServiceImpl.java:111]: Is this used in this class?

Comment (dhanendra06) [reply admin/kernel-masterdata-service/src/main/java/io/mosip/kernel/masterdata/service/impl/DeviceServiceImpl.java:111]: no

File: admin/kernel-masterdata-service/src/main/java/io/mosip/kernel/masterdata/service/impl/ZoneServiceImpl.java (hunk 1 above)
Comment (dhanendra06) [reply admin/kernel-masterdata-service/src/main/java/io/mosip/kernel/masterdata/service/impl/ZoneServiceImpl.java:74]: no

File: admin/admin-service/src/test/java/io/mosip/admin/packetstatusupdater/config/TestSecurityConfig.java [hunk 3]
```diff
@@ -4,55 +4,55 @@
...
+		httpSecurity.authorizeHttpRequests(
+				http -> http.anyRequest().permitAll());
+		httpSecurity.exceptionHandling(cfg -> cfg.authenticationEntryPoint(unauthorizedEntryPoint()));
+		httpSecurity.sessionManagement(cfg -> cfg.sessionCreationPolicy(SessionCreationPolicy.STATELESS));
+		httpSecurity.httpBasic(Customizer.withDefaults());
+		return httpSecurity.build();
 	}
 
-	@Override
+	/*@Override
```
[the remark this answers is not in the document]
Comment (dhanendra06) [reply admin/admin-service/src/test/java/io/mosip/admin/packetstatusupdater/config/TestSecurityConfig.java]: removed

File: admin/admin-service/src/main/java/io/mosip/admin/packetstatusupdater/exception/ApiExceptionHandler.java [hunk 4]
```diff
@@ -52,12 +51,18 @@ public class ApiExceptionHandler {
 	private static final Logger logger = LoggerFactory.getLogger(ApiExceptionHandler.class);
 
 
-	@ExceptionHandler(value = {AuthRestException.class})
+/*	@ExceptionHandler(value = {AuthCodeProxyExceptionHandler.class})
```
[the remark this answers is not in the document]
Comment (dhanendra06) [reply admin/admin-service/src/main/java/io/mosip/admin/packetstatusupdater/exception/ApiExceptionHandler.java]: this exception is removed from Library authflowcodeproxy so removed

**Review by LoganathanSekar7627 — APPROVED** [review]

**Review by ckm007 — APPROVED** [review]

**Pull request merged**